src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Entity
+@Data
+@Table(name = "products")
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @EqualsAndHashCode.Exclude
+    private Long id;
+    private String title;
+    private BigDecimal price;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @ToString.Exclude
+    private Category category;
```

Comment:
what about equals and hashcode? Do you need this field int these methods? Don't forget, that this field is lazy.